package.json

```diff
@@ -10,28 +10,37 @@
     "build:css": "postcss styles/tailwind.css -o styles/main.css",
     "build:watch": "postcss styles/tailwind.css -o styles/main.css --watch",
     "prepare": "husky install",
-    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha -r ts-node/register 'tests/**/*.ts' -r dotenv/config"
+    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha -r ts-node/register -r tsconfig-paths/register 'tests/**/*.ts' -r dotenv/config"
   },
   "dependencies": {
+    "@tanstack/react-query": "^4.29.3",
+    "@trpc/client": "^10.21.1",
+    "@trpc/next": "^10.21.1",
+    "@trpc/react-query": "^10.21.1",
+    "@trpc/server": "^10.21.1",
     "chai": "^4.3.7",
     "dotenv": "^16.0.3",
     "firebase": "^9.14.0",
+    "fuse.js": "^6.6.2",
     "isomorphic-unfetch": "^3.1.0",
```

Comment:
Do we need to keep this dep around?

Reply:
oops meant to highlight the `isomorphic-unfetch` dep here

Reply:
Fuse is going to be used to fuzzy search for orgs